source/_integrations/mitemp_bt.markdown

```diff
@@ -22,6 +22,9 @@ Depending on the operating system you're running, you have to configure the prop
 
 ## Configuration
 
+Note: These instructions assume either a Linux distribution or [SSH access to the machine itself](https://developers.home-assistant.io/docs/operating-system/debugging/
+).
```

Comment:
```suggestion
Note: These instructions assume either a Linux distribution or [SSH access to the machine itself](https://developers.home-assistant.io/docs/operating-system/debugging/
).
```

Developer access is not required for this, nor do we want to recommend to use that in any way.
Alternatively, the Community SSH & Web Terminal add-on can be used, which provides this functionality out of the box.

Reply:
> Community SSH & Web Terminal add-on can be used

Either I'm doing that wrong; or the hassio container I have doesn't provide access to hcitool or bluetoothctl.

![image](https://user-images.githubusercontent.com/365751/90714456-dc9f3c00-e2e6-11ea-899a-855fcea875aa.png)

I agree it would be better if the plugin supported autodiscovery; however the community based conversation threads are already directing folks towards the debugging access SSH to do this.

I'm happy to adjust this to say "won't work out of the box" and not link specifically to the SSH documentation or similar as an alternative.

Reply:
That screenshot is not the Community SSH & Web Terminal add-on...

![image](https://user-images.githubusercontent.com/195327/90714911-0b96bd00-e2a9-11ea-9632-92a1dae42903.png)

Reply:
Ah.
![image](https://user-images.githubusercontent.com/365751/90715506-98fa0180-e2e9-11ea-939d-f31c7538a77e.png)